spec/system/home_page_spec.rb

```diff
@@ -0,0 +1,53 @@
+require "rails_helper"
+
+RSpec.feature "Home Page" do
+  scenario "User visits the claims homepage" do
+    given_i_am_on_the_claims_site
+    and_i_am_on_the_start_page
+    i_can_see_the_claims_service_name_in_the_header
+    and_i_can_see_the_claims_service_value_in_the_page
+  end
+
+  scenario "User visits the placements homepage" do
+    given_i_am_on_the_placements_site
+    and_i_am_on_the_start_page
+    i_can_see_the_placements_service_name_in_the_header
+    and_i_can_see_the_placements_service_value_in_the_page
+  end
+
+  private
+
+  def given_i_am_on_the_claims_site
+    ENV["CLAIMS_HOST"] = "claims.localhost"
+    Capybara.app_host = "http://#{ENV["CLAIMS_HOST"]}"
+  end
+
+  def given_i_am_on_the_placements_site
+    ENV["PLACEMENTS_HOST"] = "placements.localhost"
+    Capybara.app_host = "http://#{ENV["PLACEMENTS_HOST"]}"
+  end
```

Comment:
### 1. Redefining environment variable

Do we need to set `ENV["CLAIMS_HOST"] =` and `ENV["PLACEMENTS_HOST"] =` here?

I see you've updated the `.env.example` file too – so I'd expect these environment variables to already be present, without needing to re-define them here.

### 2. Polluting global scope

I think we need a teardown to reset `Capybara.app_host` to `nil` again. Right now it looks like this will leak out and persist in the global scope.

We may want to consider a global setup/teardown and helper methods to set & reset the `Capybara.app_host` value, because this is likely going to be something we use across most feature specs.

Reply:
You may recall that I raised the `.env.test` issue a couple days ago. I sorted it locally by renaming my `.env.local` file to just `.env`. This makes it accessible to both the local and test environments. I think eventually we will want a `.env.test` file as there will be environment that is not shared across test and local, but this works for now.

Reply:
Ah – I've been using a `.env` file all along. I didn't realise others had named theirs `.env.local`. 🙈

Yep – all makes sense. It would definitely be nice to have environment variables for tests defined in one place, rather than manually setting them within the test scenarios.

But all good if you'd rather keep it this way for now.